tests/operators/operators.py

```diff
@@ -209,7 +241,7 @@ def test_overwrite_schema(self):
         import airflow.operators.postgres_operator
 
         sql = "SELECT 1;"
-        t = operators.postgres_operator.PostgresOperator(
+        t = airflow.operators.postgres_operator.PostgresOperator(
```

Comment:
As we're changing this anyway, I would like to import  the operator:
```
from airflow.operators.postgres_operator import PostgresOperator
```
This makes it easier to read.

Reply:
Thanks for the review @Fokko! Revised the PR.